org-ql.el

```diff
@@ -448,6 +452,29 @@ Returns cons (INHERITED-TAGS . LOCAL-TAGS)."
                  org-ql-tags-cache))
       (puthash position all-tags tags-cache))))
 
+(defun org-ql--expand-tag-groups (tags &optional exclude-groups)
+  (if (eq tags 'org-ql-nil)
+      'org-ql-nil
+    (let ((exclude-groups (append tags exclude-groups))
+          result)
+      (let (group-tags)
+        (dolist (tag tags)
+          (dolist (groups org-tag-groups-alist)
+            (when (and (not (cl-member (car groups) exclude-groups
+                                       :test 'string=))
+                       (cl-some (lambda (x)
```

Comment:
I realize it's not uncommon to do so in lambdas, but I prefer to use descriptive variable names as much as possible rather than `x`.  :)

Reply:
Also, since we already use `dash` in this package, we might as well use `--some` or `--any` here, which is more concise.

Reply:
Good point. Replaced that with `--some`. Addressed in the new commit.